azavea/district-builder-dtl-pa
# Config changes

## Overview

Make a variety of config changes as described here: https://www.pivotaltracker.com/n/projects/2140015/stories/159331237

### Checklist


## Testing Instructions

Run `./scripts/configure_pa_data` and check that all necessary changes have been made.

Closes #159331237
